jwst/extract_1d/extract1d.py

```diff
@@ -263,11 +263,11 @@ def extract1d(image, var_poisson, var_rnoise, var_flat, lambdas, disp_range,
 
             if bkg_npts == 0:
                 bkg_model = None
-                log.warning(f"Not enough valid pixels to determine background "
+                log.debug(f"Not enough valid pixels to determine background "
```

Comment:
I think this seems appropriate. It could be useful info for debugging or tracking down exactly what happened with a given spectrum, but not part of the routine logging.